### DISCUSSION TITLE
Error loading the script

### DISCUSSION THREAD

**Discussion by Ranslsad** [jstolpe/instagram-graph-api-php-sdk#6] [General]
Hello mates, i am trying to load the sdk using:
`require_once 'inc/sdk/Instagram/autoload.php'; // change path as needed`

 but it gives me the next error

```
Warning: require(er.php): Failed to open stream: No such file or directory in *****\inc\sdk\Instagram\autoload.php on line 36

Warning: require(ption.php): Failed to open stream: No such file or directory in *****\inc\sdk\Instagram\autoload.php on line 36

Fatal error: Uncaught Error: Failed opening required 'ption.php' (include_path='E:\xampp\php\PEAR') in *****\inc\sdk\Instagram\autoload.php:36 Stack trace: #0 *****\kumbia\bootstrap.php(38): {closure}('KumbiaException') #1 [internal function]: handle_exception(Object(Error)) #2 {main} thrown in *****\inc\sdk\Instagram\autoload.php on line 36
```

**Comment by jstolpe** [comment]
hey, thanks for checking out the sdk. Make sure your paths are correct and looking in the correct places. From those errors I am guessing something is not quite right with the folder structure and setup on your server. If you can try using composer to install the package and then include that autoloader.

**Comment by AguCollomb** [comment]
Hello, you must put the complete directori in require_once function.
